docker/Dockerfile.dev

```diff
@@ -7,15 +7,20 @@ WORKDIR /src
 
 
 # System packages ##############################################################
-RUN apt-get update --fix-missing
-RUN apt-get -y install curl
-RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
-RUN apt-get -y install nodejs python python-dev \
-    fish man \
-    python-pip gcc libpq-dev ffmpeg imagemagick unzip \
+
+RUN apt-get update --fix-missing && apt-get -y install \
+    curl fish man \
+    python python-dev python-pip \
+    gcc libpq-dev ffmpeg imagemagick unzip \
     ghostscript python-tk make git gettext openjdk-9-jre-headless libjpeg-dev \
     wkhtmltopdf fonts-freefont-ttf xfonts-75dpi poppler-utils
+
+# Download and install wkhtmltox
 RUN curl -L -o wkhtmltox.deb https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.xenial_amd64.deb && dpkg -i wkhtmltox.deb
+
+# Download then install node
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - &&\
+    apt install -y nodejs
```

Comment:
I know `apt` is the same as `apt-get` but maye use apt-get here too to be consistent